classes/03class/exercises/c03-docker06/christina-roperto/c03-docker06.txt

```diff
@@ -0,0 +1,5 @@
+Building: 
+$ docker build -t hello-name:v1 .
+
+Running by passing environment variables (https://docs.docker.com/engine/reference/commandline/run/): 
+$ docker run --rm -it -e "name=Christina Roperto" hello-name:v1
```

Comment:
Can you do this without using environment variables? In a way that by running
`docker run --rm 0it hello-name:v1 Christina` will output:
`Hello Christina`
?

Reply:
Done!